contracts/prize-strategy/multiple-winners/MultipleWinners.sol

```diff
@@ -40,6 +54,30 @@ contract MultipleWinners is PeriodicPrizeStrategy {
     _setNumberOfWinners(_numberOfWinners);
   }
 
+  function setBlocklisted(address _user, bool _blocklist) external onlyOwner requireAwardNotInProgress returns (bool) {
+    isBlocklisted[_user] = _blocklist;
+
+    emit BlocklistSet(_user, _blocklist);
+
+    return true;
+  }
+
+  function setCarryBacklist(bool _carry) external onlyOwner requireAwardNotInProgress returns (bool) {
```

Comment:
should this be `setCarry**Block**List()`?

Reply:
Updated function naming when adding tests.